test/Reddit.spec.ts

```diff
@@ -0,0 +1,120 @@
+    let wallets: any;
+    let Reddit: any;
+    let User: any;
+    let testTokenInstance;
+    let depositManagerInstance;
+    let rollupCoreInstance: any;
+    let IMTInstance: any;
+    before(async function () {
+        depositManagerInstance = await DepositManager.deployed();
+        wallets = walletHelper.generateFirstWallets(walletHelper.mnemonics, 10);
+        Reddit = {
+            Address: wallets[0].getAddressString(),
+            Pubkey: wallets[0].getPublicKeyString(),
+            Amount: 50,
+            TokenType: 1,
+            AccID: 2,
+            Path: "2",
+            nonce: 0,
+        };
+        User = {
+            Address: wallets[1].getAddressString(),
+            Pubkey: wallets[1].getPublicKeyString(),
+            Amount: 10,
+            TokenType: 1,
+            AccID: 3,
+            Path: "3",
+            nonce: 0,
+        };
+        testTokenInstance = await utils.registerToken(wallets[0]);
+        await testTokenInstance.transfer(Reddit.Address, 100);
+        await depositManagerInstance.depositFor(
+            Reddit.Address,
+            Reddit.Amount,
+            Reddit.TokenType,
+            Reddit.Pubkey
+        );
+        rollupCoreInstance = await RollupCore.deployed();
+        IMTInstance = await IMT.deployed();
+    })
+    it("Should Create Account for the User", async function () {
+        const createAccountInstance = await createAccount.deployed();
+        // Call to see what's the accountID
+        const accountId = await createAccountInstance.createPublickeys.call([User.Pubkey]);
+        assert.equal(accountId.toString(), "3");
+        // Actual execution
+        await createAccountInstance.createPublickeys([User.Pubkey]);
+
+        const tx = {
+            toIndex: 3,
+            tokenType: 1
+        } as CreateAccount;
+        const RollupUtilsInstance = await RollupUtils.deployed();
+        const signBytes = await RollupUtilsInstance.getCreateAccountSignBytes(tx.toIndex, tx.tokenType);
+        tx.signature = utils.sign(signBytes, wallets[0]);
+        const balanceRoot = await rollupCoreInstance.getLatestBalanceTreeRoot();
+        const accountRoot = await IMTInstance.getTreeRoot();
+        const ZeroAccount: Account = {
+            ID: 0,
+            tokenType: 0,
+            balance: 0,
+            nonce: 0,
+            burn: 0,
+            lastBurn: 0
+        }
+        const RedditAccountLeaf = await utils.createLeaf(Reddit)
+        const zeroHashes = await utils.defaultHashes(4);
+        const coordinator_leaves = await RollupUtilsInstance.GetGenesisLeaves();
+        const NewAccountSiblings: Array<string> = [
+            RedditAccountLeaf,
+            utils.getParentLeaf(coordinator_leaves[0], coordinator_leaves[1]),
+            zeroHashes[2],
+            zeroHashes[3],
+        ];
+        const NewAccountMP = {
+            accountIP: {
+                pathToAccount: User.Path,
+                account: ZeroAccount,
```

Comment:
The account field here can be an arbitrary account. We don't validate it since we validate zero account "leaf" only. We just need to pass something here because the input struct requires it.